### ISSUE TITLE
Message repeating in access serviice ,Sent messages are repeating while sending to firebase

### ISSUE THREAD

**Issue by akshay-nayak** [patanjalikr13/Android-Undetectable-Spy-App#6]

**Comment by patanjalik** [comment]
Yes, It reads whole screen at once. It is tough to differentiate from already pushed messages from new ones. Besides, I didn't wanted to store whole copy of RTDB on local device to avoid heavy RAM or battery usage in order to maintain stealthiness.
